CIP-0087/CIP-0087.md

```diff
@@ -0,0 +1,554 @@
+---
+CIP: 0087
```

Comment:
```suggestion
CIP: ?
```
Choosing a CIP number in advance of a review is always a problem (collisions with CIPs in progress & sometimes online discussions are almost assured) as well as a specific problem in this case:
- https://github.com/cardano-foundation/CIPs/pull/440

Please @kozross also make a link to the readable version of your proposal in your OP (https://github.com/cardano-foundation/CIPs/pull/624#issue-2005257354) after you've
- renamed your containing directory to something without a number (e.g. `CIP-XXXX` or `CIP-Integer-ByteString`). 
- like all current CIPs, renamed the file itself to `README.md`